packages/formation-react/src/components/ErrorableSelect/ErrorableSelect.jsx

```diff
@@ -144,26 +144,26 @@ ErrorableSelect.propTypes = {
   required: PropTypes.bool,
 
   /**
-   * is there an empty selectable thing
+   * Is there an empty selectable option
    */
   includeBlankOption: PropTypes.bool,
 
   /**
-   * Description that shows up for the blank option, when includeBlankOption is true
+   * Description that shows up for the blank option, when `includeBlankOption` is true
    */
   emptyDescription: PropTypes.string,
 
-  /** `value` - object containing:
+  /** Object containing:
    *   - `value`: Value of the select field.
-   *   - `dirty`: boolean. Whether a field has been touched by the user.
+   *   - `dirty`: Whether a field has been touched by the user.
```

Comment:
To get these to show up as multiple lines in Storybook's `Docs` page, you'll want a blank line between these. Like Markdown.

Reply:
Thank you!